Tools/SandboxTest.ps1

```diff
@@ -0,0 +1,110 @@
+# Parse Arguments
+
+param ([string] $Manifest = $(throw "The Manifest parameter is required."))

```

Comment:
I think `[Parameter(Mandatory=$true)]` is the canonical way to mark parameters as required.

Reply:
Awesome!!! Great to know!

Reply:
Implemented!

Reply:
But sadly this made the script to always print a blank newline at the beginning. I didn't find how to prevent that.